cfg/gnu.cfg

```diff
@@ -47,6 +47,7 @@
   <define name="__builtin_nanl(str)" value="nanl(str)"/>
   <define name="__builtin_signbit(value)" value="signbit(value)"/>
   <define name="__extension__" value=""/>
+  <define name="__typeof__" value="typeof"/>
```

Comment:
we can make such replacement a bit stricter..

    <define name="__typeof__(T)" value="typeof(T)"/>

I do not know if that matters in practice but that feels a bit nicer.

Reply:
Yes, I will change it

Reply:
```suggestion
  <define name="__typeof__(T)" value="typeof(T)"/>
```